app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -166,54 +166,25 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
 
     override fun onBackPressed() {
         val fragmentManager = supportFragmentManager
+
+        val maybeBrowserFragment = (fragmentManager.findFragmentByTag(WebRenderFragment.FRAGMENT_TAG)
+                as WebRenderFragment?)?.let {
+            if (it.isVisible) it else null
```

Comment:
I know we're not doing MVVM in this branch, but could we add a TODO here?  I want to make sure we don't base this logic on view visibility after the refactor.